### DISCUSSION TITLE
[ios] change branches in repo

### DISCUSSION THREAD

**Discussion by AndrewSB** [community/community#3707] [Mobile]
while browsing code, it would be great to be able to change branches/see branches.
i has to switch to web to be able to do this

**Comment by brianlovin** [comment]
Thanks for the feedback – it's possible to switch branches in the app now, but it does require you to navigate back to the root repo profile view. The branch switcher is above the "browse code" entry point. Switching branches while browsing the code at some arbitrary point in the file tree is a neat idea, but technically complex, so we'll have to scope this kind of work out for the future. Hopefully the current branch switcher can help in the meantime 😊

**Reply by AndrewSB** [reply]
oh got it! i didn't see that, i'll look again
